casbin/core_enforcer.py

```diff
@@ -181,6 +181,8 @@ def load_policy(self):
         self.model.clear_policy()
         self.adapter.load_policy(self.model)
 
+        self.model.sort_policies_by_priority()
```

Comment:
@ffyuanda  Please refer to the latest version :https://github.com/casbin/casbin/blob/master/enforcer.go#L281